src/tic/t/tic.t

```diff
@@ -160,20 +156,18 @@ hi
 --- http_config
 init_by_lua_block {
   tic = require "tic"
-  iputils = require "resty.iputils"
+  cidr = require "libcidr-ffi"
 }
 --- more_headers eval
 use MIME::Base64;
 "Authorization: Bearer header." . encode_base64('{"email":"hi@gsa.gov"}')
 --- config
 location = /t {
   access_by_lua_block {
-    cidrs, _ = iputils.parse_cidrs({"10.0.0.0/24"})
-    whitelist={["gsa.gov"]=cidrs}
+    whitelist={["gsa.gov"]={cidr.from_str("10.0.0.0/24")}}
```

Comment:
pedantic, but if you are going to use rfc3849 addresses for v6 tests, should use rfc5737 addresses for v4?